profiler/distributions.py

```diff
@@ -0,0 +1,134 @@
+from __future__ import absolute_import, division, print_function
+
+import argparse
+
+import torch
+from torch.autograd import Variable
+
+from profiler.profiling_utils import Profile, profile_print
+from pyro.distributions import (bernoulli, beta, categorical, cauchy, delta, dirichlet, exponential, gamma, halfcauchy,
+                                lognormal, normal, poisson, uniform)
+
+
+def T(arr):
+    return Variable(torch.Tensor(arr))
+
+
+TOOL = 'timeit'
+TOOL_CFG = {}
+DISTRIBUTIONS = {
```

Comment:
itd be nice to get this from tests/distributions/conf.py

Reply:
I tried to see if there was some way to share fixtures / test cases, but opted for a more decoupled code in the end. Maybe we can build a pytest plugin that does this too, so that we have all the flexibility of parameterized fixtures, but right now it seemed like taking on much more complexity than what we need.

Reply:
I agree it's cleaner to decouple them, so we can choose examples differently in the two cases: test corner cases vs profile numerically consistent cases.